common/src/loki_logger.cpp

```diff
@@ -63,6 +63,7 @@ void init_logging(const std::string& data_dir,
 
     auto logger = std::make_shared<spdlog::logger>("loki_logger", sinks.begin(),
                                                    sinks.end());
+    logger->set_level(log_level);
```

Comment:
already in #208

Reply:
Yes I manually merge that change from `main.cpp` into `loki_logger.cpp`

Reply:
Why not just rebase?

Reply:
I'm not sure to follow.
I have rebased this PR onto dev, but since I moved the logger initialisation code from `main.cpp` to `loki_logger.cpp`, there was a conflict. I resolved itt by adding the line above in `loki_logger.cpp`.

Reply:
OK, my bad then.